spynnaker/pyNN/models/neuron/synapse_dynamics/abstract_synapse_dynamics.py

```diff
@@ -65,45 +65,29 @@ def get_delay_maximum(self, connector):
         """
         return connector.get_delay_maximum()
 
-    def get_delay_variance(
-            self, connector, n_pre_slices, pre_slice_index, n_post_slices,
-            post_slice_index, pre_vertex_slice, post_vertex_slice):
+    def get_delay_variance(self, connector):
```

Comment:
A few things to consider here in conjunction with https://github.com/SpiNNakerManchester/sPyNNaker/pull/525 - but again, not enough to stop this from being merged now.  I'd be happy to do the merge from this PR into that one if necessary.